lms/static/js/fixtures/ccx/schedule.html

```diff
@@ -1,83 +1,39 @@
-<div class="ccx-schedule-container">
-  <div id="ccx-schedule"></div>
-  <div id="new-ccx-schedule"></div>
-</div>
+<%block name="jsextra">
+  <script src="${static.url('js/vendor/backbone-min.js')}"></script>
+  <script src="${static.url('js/vendor/timepicker/jquery.timepicker.js')}"></script>
+  <link rel="stylesheet" type="text/css" href="${static.url('js/vendor/timepicker/jquery.timepicker.css')}" />
+  <style>
+    .ui-timepicker-list { z-index: 100000; }
+    .ui-datepicker { z-index: 100000 !important; }
+    input.date, input.time { width: auto !important; display: inline !important; }
+  </style>
+</%block>
 
-<section id="enter-date-modal" class="modal" aria-hidden="true">
-  <div class="inner-wrapper" role="dialog">
+<section id="enter-date-modal" class="modal"
+         tabindex="-1" role="dialog" aria-labelledby="ccx_schedule_set_date_heading">
+  <div class="inner-wrapper">
     <button class="close-modal">
-      <i class="fa-remove"></i>
-      <span class="sr">
-        Close
-      </span>
+      <i class="icon fa fa-remove" aria-hidden="true"></i>
+      <span class="sr">${_("Close")}</span>
     </button>
     <header>
-      <h2></h2>
+      <h2 id="ccx_schedule_set_date_heading"></h2>
     </header>
-    <form role="form">
+    <form class="ccx-form">
       <div class="field datepair">
-        <label></label>
-        <input placeholder="Date" class="date" type="text" name="date"/ size="11">
-        <input placeholder="Time" class="time" type="text" name="time"/ size="6">
+        ## Translators: This explains to people using a screen reader how to interpret the format of YYYY-MM-DD
```

Comment:
This is a test fixture file. There is no need to translate strings for test files, and please do not do this if the strings differ at all from production strings (just adds extra work for translators).